api/api-referential/referential-internal/src/main/java/fr/gouv/vitamui/referential/internal/server/profile/ProfileInternalService.java

```diff
@@ -301,4 +302,13 @@ private ProfileResponseDto findAll(VitamContext vitamContext, JsonNode query) {
             throw new InternalServerException("Unable to find archivalProfiles", e);
         }
     }
+
+    public boolean check(VitamContext vitamContext, ProfileDto profileDto) {
+        try {
+            ProfileDto existingProfileDto = getOne(vitamContext, profileDto.getIdentifier());
+            return Objects.nonNull(existingProfileDto);
+        } catch (NotFoundException notFoundException) {
+            return false;
```

Comment:
Logger le msg de l'exception

Reply:
je ne la fait pas : si y a not found il existe pas , si null est retourné il existe pas